rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -857,7 +857,7 @@ void trophy_manager_dialog::PopulateTrophyTable()
 			}
 		}
 
-		const QString unlockstate = data->trop_usr->GetTrophyUnlockState(trophy_id) ? tr("Unlocked") : tr("Locked");
+		const QString unlockstate = data->trop_usr->GetTrophyUnlockState(trophy_id) ? tr("Earned") : tr("Not Earned");
```

Comment:
Why earned and not earned?

Reply:
As noted in the issue, Earned / Not Earned aligns with Sony 'speak'.  Unlocked and Locked does not.